Update http doc intro (#37)
liteflow-labs/docs: services/http.md

The HTTP Service lets you create an HTTP endpoint with a unique URL that triggers the associated process when an HTTP request is received.

> You can also use this service to call any endpoint from within your processes, other HTTP endpoints created with Liteflow, or any external endpoint.

Reviewer: ```suggestion
You can also use this service to call any endpoints from within your processes. They can be HTTP endpoints created with Liteflow but also any external endpoints (webhooks).
```
